ApplicationLibCode/ProjectDataModel/GridCrossPlots/RimGridCrossPlotDataSet.cpp

```diff
@@ -1421,7 +1423,12 @@ void RimGridCrossPlotDataSet::triggerPlotNameUpdateAndReplot()
     {
         parent->updateCurveNamesAndPlotTitle();
         parent->reattachAllCurves();
-        parent->updateConnectedEditors();
+
+        // updateConnectedEditors() causes the UI editor to be recreated (and the widgets contained in the UI editor). The function
+        // calling this function can in some cases comes from the widget to be destroyed, typically the tree view selection editor
+        // displaying available result properties. Using a scheduler will ensure that all other slots are processed before the
+        // updateConnectedEditors() is called.
+        caf::UpdateEditorsScheduler::instance()->scheduleUpdateConnectedEditors( parent );
```

Comment:
would it be better to hide the scheduler a bit in a method like:
parent->scheduleUpdateConnectedEditors();

Reply:
Agree, will fix